Session/Media Viewing & Editing/MessageInfoScreen.swift

```diff
@@ -305,7 +306,10 @@ struct MessageInfoScreen: View {
                                     Button(
                                         action: {
                                             actions[index].work()
-                                            dismiss()
+                                            feedbackMessage = actions[index].feedback
+                                            DispatchQueue.main.asyncAfter(deadline: .now() + 2, execute: {
```

Comment:
Looks like at the moment if `feedback` is an empty string then clicking on an action results in an empty toast being shown and a 2 second delay before returning to the previous screen - might be good to skip the delay if there is no `feedback` (eg. `Reply` and `Delete` both behave a bit strangely right now)